prometheus/rootfs/bin/boot.sh

```diff
@@ -22,18 +22,23 @@ done
 
 set +e
 
-OPTS=""
+until [ -n "${DEIS_MONITOR_ALERT_SERVICE_HOST+1}" ]; do
```

Comment:
i thought about that but given that the namespace is not static and the namespace is part of the dns name how can you anticipate ?  can we just assume the default namespace for now ?